.gitignore

```diff
@@ -11,8 +11,6 @@ test/.project
 *.lock
 .idea/
 
-dist/
```

Comment:
I actually want to ignore the dist folder:
* Different versions are already available in the AppStore
* The versions are available on Github as releases